scss/_settings_colors.scss

```diff
@@ -12,21 +13,20 @@ $color-mid: #999 !default;
 $color-mid-dark: #666 !default;
 $color-dark: #111 !default;
 $color-x-dark: #000 !default;
-
-$color-negative: #c7162b !default;
-$color-caution: #f99b11 !default;
-$color-positive: #0e8420 !default;
-$color-information: #335280 !default;
+$color-input-shadow: rgba($color-x-dark, 0.12) !default;
 
 $color-label-validated: #006b75;
 
-$color-input-shadow: rgba($color-x-dark, 0.12) !default;
-
-$color-accent: $color-brand !default;
-$color-accent-background: $color-accent !default;
+$color-negative: #c7162b !default;
+$color-caution: #f99b11 !default;
+$color-positive: hsl(129, 81%, 29%) !default;
 
-// for visual outline on :active / :focus elements
-$color-focus: #19b6ee !default;
+$color-link: #06c !default;
+$color-link-visited: #7d42b8 !default;
```

Comment:
Is this the visitied color we want to use now? `#7d42b8`

Reply:
This new visited links look very different?
![image](https://user-images.githubusercontent.com/1413534/81579960-b9d6c080-93a4-11ea-9a4a-b3e22ae27cde.png)

Reply:
yes, turns out there are requirements for contrast ratio between link states. There's a good article here: https://webaim.org/blog/wcag-2-0-and-link-colors/